src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -107,6 +110,51 @@ public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager
             return Ok(status);
         }
 
+        [HttpGet]
```

Comment:
Regarding security, having this be GET will allow Readers to list this info, which is OK because the Kudu APIs for listing processes are also allowed for Readers as well. Just pointing this out so we keep this in mind that we don't extend this API to include any secrets - if we did, we'd have to lock it down to only Contributors.

Reply:
That is a good point. Let me add a comment to the method outlining this information..